src/actions.cr

```diff
@@ -1,38 +1,21 @@
 require "kemal"
-
-macro fmt_route(route = "", namespace = false)
-  {{ "/" + @type.stringify
-       .downcase
-       .gsub(/^DppmRestApi::Actions::/, "")
-       .gsub(/::/, "/") }} {% if namespace %} + "/:namespace" {% end %} + ( {{route}} || "")
-end
-
-# the "last" or -1st block argument is selected because context is always the
-# argument -- either the only or the second of two
-{% for method in %w(get post put delete ws options) %}
-macro relative_{{method.id}}(route, &block)
-  {{method.id}} fmt_route(\{{route}}) do |\{{block.args.splat}}|
-    namespace = \{{block.args[-1]}}.params.query["namespace"]? || DPPM::Prefix.default_group
-    \{{block.body}}
-  end
-end
-{% end %}
-
 require "./access"
+require "./actions/utils"
```

Comment:
It would be more logical to include `utils` where it is used, in each action file.

Reply:
Utils are inherently tools that are there for all actions, so it makes more sense to me for it to be here, but if you feel strongly about it I can change it.

Reply:
If that's for all actions, it make sense.
So, I don't think the utils name is appropriate, maybe `RouteHelper`

Reply:
Sounds good.